### PR TITLE
fix(Popup): update global payment toggle UI

### LINKED ISSUE #787: [UX feedback]: global payment toggle
Based on feedback during yesterday's team meeting, update global payment toggle feature's UI.

- Remove the large power button on Home screen when disabled. Use `NotMonetized` component to display message.
- Continue showing the button in Header, don't remove it to show the large button.
- Show the default colored icon in toolbar, instead of disabled gray one.

### PR DESCRIPTION
**Pull request by DarianM**
## Context

Closes #787 .

## Changes proposed in this pull request

- Removed the large power button on Home screen when disabled, display disable message only.
- Always showing the toggle button in Header.
- Show the default colored icon in toolbar, instead of disabled gray one.

### CODE REVIEW

**Review by sidvishnoi — APPROVED** [review]
LGTM.
Maybe the toggle's `input` should've `onClick={(ev) => ev.currentTarget.blur()}` so the focus styles get removed?

File: src/pages/popup/components/TogglePaymentsButton.tsx [hunk 1]
```diff
@@ -3,14 +3,10 @@ import { Power } from '@/pages/shared/components/Icons';
 import { cn } from '@/shared/helpers';
 import { useMessage } from '@/popup/lib/context';
 import { dispatch } from '@/popup/lib/store';
+import { usePopupState } from '@/popup/lib/store';
```
Comment (sidvishnoi) [inline src/pages/popup/components/TogglePaymentsButton.tsx:6]:
```suggestion
import { useMessage, usePopupState } from '@/popup/lib/context';
import { dispatch } from '@/popup/lib/store';
```

Comment (sidvishnoi) [inline src/pages/popup/components/TogglePaymentsButton.tsx:6]:
```suggestion
import { dispatch, usePopupState } from '@/popup/lib/store';
```

Comment (DarianM) [reply src/pages/popup/components/TogglePaymentsButton.tsx:6]: should we add eslint to enforce a single import statement for modules with the same path? [no-duplicate-imports](https://eslint.org/docs/latest/rules/no-duplicate-imports)

**Pull request merged**